### PR TITLE
BRD - UI: feat - Add business to 'My List'

### PR DESCRIPTION
**Pull request by deetz99**
issue: bcGov/entity#22572

- modal to add business to your business registry list
- loading/no options/form/error/magic link sent modal states

https://github.com/user-attachments/assets/e0028ae3-313f-4d5c-9b22-03ae5e6bd6fe

### Todo: unit test

### CODE REVIEW

File: business-registry-dashboard/app/app.config.ts
Comment (deetz99) [inline business-registry-dashboard/app/app.config.ts:1]: update config styling for accordian trigger button, form group label and help, UInput error variant

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-registry-dev--pr-22-69s8dmyz.web.app

**Comment by deetz99** [comment]
Still an issue with the api's in the temp url

File: business-registry-dashboard/app/components/Form/AddBusiness/EmailAuthSent.vue
Comment (deetz99) [inline business-registry-dashboard/app/components/Form/AddBusiness/EmailAuthSent.vue:1]: displays the 'magic link sent' modal state

File: business-registry-dashboard/app/components/Form/AddBusiness/Error.vue
Comment (deetz99) [inline business-registry-dashboard/app/components/Form/AddBusiness/Error.vue:1]: error state based off request type and error status code

File: business-registry-dashboard/app/components/Form/AddBusiness/index.vue
Comment (deetz99) [inline business-registry-dashboard/app/components/Form/AddBusiness/index.vue:1]: wrapper to display different form states/handle transition

File: business-registry-dashboard/app/components/Form/AddNameRequest/Base.vue
Comment (deetz99) [inline business-registry-dashboard/app/components/Form/AddNameRequest/Base.vue:1]: this is set by 'setScreenReaderAlert' so the extra assignment is unnecessary

File: business-registry-dashboard/app/components/Modal/Base.vue
Comment (deetz99) [inline business-registry-dashboard/app/components/Modal/Base.vue:1]: add fullscreen changes to base modal component

File: business-registry-dashboard/app/components/Modal/ManageBusiness.vue
Comment (deetz99) [inline business-registry-dashboard/app/components/Modal/ManageBusiness.vue:1]: wrapper modal for add business form, handles auth options, will display loading/no option or form state

File: business-registry-dashboard/app/composables/useBrdModals.ts
Comment (deetz99) [inline business-registry-dashboard/app/composables/useBrdModals.ts:1]: open manage business modal using composable

File: business-registry-dashboard/tests/e2e/utils/auth-setup.ts
Comment (deetz99) [inline business-registry-dashboard/tests/e2e/utils/auth-setup.ts:1]: fix lint error

File: business-registry-dashboard/app/components/Modal/ManageBusiness.vue [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+})
+
+const enableDelegationFeature = computed(() => {
+  // return LaunchDarklyService.getFlag(LDFlags.EnableAffiliationDelegation) || false // TODO: fix after adding launch darkly
+  return true
+})
+
+const showPasscodeOption = computed(() => {
+  // const allowableBusinessPasscodeTypes: string = LaunchDarklyService.getFlag(LDFlags.AllowableBusinessPasscodeTypes) || 'BC,SP,GP' // TODO: implememnt after adding launch darkly
+  const allowableBusinessPasscodeTypes: string = 'BC,SP,GP'
```
Comment (JazzarKarim) [inline business-registry-dashboard/app/components/Modal/ManageBusiness.vue:39]: I'm guessing we probably don't need to indicate here the string type (`:string = `)?

**Review by JazzarKarim — APPROVED** [review]
Looks great! 👍  Learned a lot here, thanks!

File: business-registry-dashboard/app/components/Modal/ManageBusiness.vue (hunk 1 above)
Comment (deetz99) [reply business-registry-dashboard/app/components/Modal/ManageBusiness.vue:39]: For these simpler objects the type is inferred pretty well, maybe when I add in the launchdarkly stuff I'll need to update it.

**Review by thorwolpert — APPROVED** [review]
I really like this - $authApi
the more I see it, the more I like it.

**Pull request merged**